### PR TITLE
Adds SK Telemetry using Semantic Conventions for AI

### PR DESCRIPTION
**Pull request by luisquintanilla**

### CODE REVIEW

**Comment by luisquintanilla** [comment]
cc: @stephentoub

File: src/WebApp/Program.cs [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
+// Configure SK Telemetry
+builder.Services.AddOpenTelemetry()
+    .WithMetrics(m =>
+    {
+        m.AddMeter("Microsoft.SemanticKernel*");
+    })
+    .WithTracing(t =>
+    {
+        t.AddSource("Microsoft.SemanticKernel*");
+    });
```
Comment (stephentoub) [inline src/WebApp/Program.cs:22]: Should all of these changes go into service defaults where the rest of telemetry is configured?

Comment (luisquintanilla) [reply src/WebApp/Program.cs:22]: Good question. We can do it there as well, but AFAIK we're mainly surfacing the information from the web app. Not sure if we'd get additional info if it was baked into service defaults.

**Comment by luisquintanilla** [comment]
Closing and reopening a new branch based on main.

**Pull request closed without merging**